core/domain/stats_jobs_one_off.py

```diff
@@ -401,8 +401,7 @@ def reduce(exp_id, stringified_values):
                         # potential destination from the initial state. For
                         # these states, the first hit count is the completions
                         # count of the exploration.
-                        if dest_state == 'END' and dest_state not in (
-                                versioned_exploration.states):
+                        if dest_state == 'END':
```

Comment:
Please explain this change.

Reply:
@seanlip While iterating through the answer group outcomes leading out from the initial state, if the state is an END state, we append to our list the num_completions, otherwise, we append to our list, the first_hit_count.